### DISCUSSION TITLE
Cross referencing equations

### DISCUSSION THREAD

**Discussion by Yue-Zhengyuan** [KaTeX/KaTeX#3708] [Q&A] [answered]
Is it possible to cross-reference equations in HTML/Markdown with KaTeX? In LaTeX, this is done by assigning a label by `\label{xxx}`, and then refer to the equation by `\ref{xxx}`. But it seems now that both `\label` and `\ref` are not implemented. But it is also OK if there is some advanced HTML trick that can do the same thing.

**Comment by edemaine** [comment, marked as answer]
There is no built-in way currently, but it's on the list. See #2798 and #2003 for relevant discussion/plans and some current workarounds.
